TranslatorReasonerAPI.yaml

```diff
@@ -470,46 +470,75 @@ components:
     Attribute:
       type: object
       description: >-
-        Generic attribute for a node or an edge that expands key-value pair
-        concept by including a type of this attribute from a suitable ontology,
-        a source of this attribute, and (optionally) a url with additional
-        information about this attribute.
+        Generic attribute for a node or an edge that expands the key-value
+        pair concept by including fields for additional metadata. These fields
+        can be used to describe the source of the statement made in key-value
+        pair of the attribute object, or describe the attribute's value itself
+        including its semantic type, or a url providing additional information
+        about it.
       properties:
-        name:
+        attribute_type_id:
+          $ref: '#/components/schemas/CURIE'
+          description: >-
+            The 'key' of the attribute object, holding a CURIE of an ontology
+            property defining the attribute (preferably the CURIE of a
+            Biolink association slot). This property captures the relationship
+            asserted to hold between the value of the attribute, and the node
+            or edge from  which it hangs. For example, that a value of
+            '0.000153' represents a p-value supporting an edge, or that
+            a value of 'ChEMBL' represents the original source of the knowledge
+            expressed in the edge.
+          example: Biolink:has_p-value_evidence, Biolink:has_original_source
```

Comment:
These do not appear to be proper biolink CURIEs.

Reply:
I also cannot find these examples in the biolink data model. Are they perhaps ones that have not made it into the documentation?

Reply:
Yes, these are examples of edge properties proposed to support capture of EPC in Attributes, but are not yet in Biolink.  Vetting and getting these into Biolink is our next step to support use of the Attribute model for consistent and computable representation of edge metadata.

Reply:
If these slots are coming to biolink, that's fine, but we've been using the prefix (lowercase) "biolink". Is that correct?